app/javascript/pages/admin/IndividualEvents/index.js

```diff
@@ -3,82 +3,168 @@ import { compose, graphql } from 'react-apollo'
-    </tr>
-  ))
+class IndividualEvents extends React.Component {
+  state = {
+    selected: [],
+    bulkAction: null,
+    selectAllChecked: false,
+  }
 
-const IndividualEvents = ({
-  data: { networkStatus, pendingIndividualEvents },
-  approveIndividualEvent,
-  rejectIndividualEvent,
-}) =>
-  networkStatus === NetworkStatus.loading ? (
-    <Loading />
-  ) : (
-    <div>
-      <h3>Pending Approval</h3>
-      <table className={s.table}>
-        <thead>
-          <tr>
-            <th>User</th>
-            <th>Description</th>
-            <th>Date</th>
-            <th>Duration (min)</th>
-            <th>Organization</th>
-            <th>Type</th>
-            <th>Actions</th>
-          </tr>
-        </thead>
-        <tbody>{buildRows(pendingIndividualEvents, rejectIndividualEvent, approveIndividualEvent)}</tbody>
-      </table>
-    </div>
-  )
+  individualEventsColumns = [
+    {
+      id: 'selection',
+      Header: props => (
+        <input
+          checked={this.state.selectAllChecked}
+          type="checkbox"
+          onClick={e => {
+            const { pendingIndividualEvents } = this.props.data
+            if (this.state.selectAllChecked) {
+              this.setState({ selected: [], selectAllChecked: false })
+            } else {
+              this.setState({ selected: pendingIndividualEvents, selectAllChecked: true })
+            }
+          }}
+        />
+      ),
+      Cell: props => (
+        <input
+          type="checkbox"
+          checked={this.state.selected.includes(props.value)}
+          onClick={e => {
+            if (e.target.checked) {
+              this.setState(prevState => ({ selected: [...prevState.selected, props.value] }))
+            } else {
+              this.setState(prevState => ({ selected: [...prevState.selected].filter(v => v.id != props.value.id) }))
+            }
+          }}
+        />
+      ),
+      width: 50,
+      style: { textAlign: 'center' },
+      sortable: false,
+      accessor: d => d,
+    },
+    {
+      id: 'user',
+      Header: 'User',
+      accessor: 'user.name',
+    },
+    {
+      id: 'description',
+      Header: 'Description',
+      accessor: 'description',
+    },
+    {
+      id: 'date',
+      Header: 'Date',
+      Cell: props => <span>{moment(props.value).format('MMMM D, YYYY')}</span>,
+      accessor: 'date',
+    },
+    {
+      id: 'duration',
+      Header: 'Duration (min)',
+      accessor: 'duration',
+    },
+    {
+      id: 'organization',
+      Header: 'Organization',
+      accessor: 'organization.name',
+    },
+    {
+      id: 'type',
+      Header: 'Type',
+      accessor: 'eventType.title',
+    },
+  ]
+
+  render() {
+    const { data } = this.props
+    const { networkStatus, pendingIndividualEvents } = data
+    return networkStatus === NetworkStatus.loading ? (
+      <Loading />
+    ) : (
+      <div className={s.eventsTable}>
+        <h3>Pending Approval</h3>
+        <div>
+          <select
```

Comment:
could use Garden's select here?

Reply:
I don't think we have garden added to this project yet, it's a good idea though. I'd be happy to do a PR later on to redo components to garden components. All the UI isn't too consistent either, forms etc. So might bundle those changes later on.

It's an admin interface so I feel it should be fine for now.